moped-database/migrations/default/1759785362249_ecapris_funding/up.sql

```diff
@@ -0,0 +1,38 @@
+-- Add funding sync tracking to projects table
+ALTER TABLE moped_project
+ADD COLUMN should_sync_ecapris_funding BOOLEAN DEFAULT FALSE;
+
+COMMENT ON COLUMN moped_project.should_sync_ecapris_funding IS 'Indicates if project funding should be synced from eCAPRIS';
+-- Fix eCAPRIS name in existing comment
+COMMENT ON COLUMN moped_project.should_sync_ecapris_statuses IS 'Indicates if project statuses should be synced from eCAPRIS';
+
+-- Update moped_proj_funding table
+ALTER TABLE moped_proj_funding
+ADD COLUMN ecapris_funding_id INTEGER,
+ADD COLUMN is_legacy_funding_record BOOLEAN DEFAULT FALSE,
+ADD COLUMN is_editable BOOLEAN GENERATED ALWAYS AS (ecapris_funding_id IS NULL) STORED,
+ADD COLUMN fdu TEXT DEFAULT NULL,
```

Comment:
We currently have a generated column called `fund_dept_unit` that pulls the F and DU parts out of `fund` and `dept_unit` json columns, but this new `fdu` column will store the FDU string that we can tie back to eCAPRIS by the unique `ecapris_funding_id`.